src/app-layer-ssl.h

```diff
@@ -176,9 +178,17 @@ typedef struct SSLState_ {
 
     /* there might be a better place to store this*/
     uint16_t hb_record_len;
+
+    uint8_t finished;
+
+    DetectEngineState *de_state;
+
+    uint16_t events;
```

Comment:
<pre>
struct SSLState_ {
public:

	Flow *                     f;                    /*     0     8 */
	uint32_t                   flags;                /*     8     4 */

	/* XXX 4 bytes hole, try to pack */

	SSLStateConnp *            curr_connp;           /*    16     8 */
	SSLStateConnp              client_connp;         /*    24   120 */
	/* --- cacheline 2 boundary (128 bytes) was 16 bytes ago --- */
	SSLStateConnp              server_connp;         /*   144   120 */
	/* --- cacheline 4 boundary (256 bytes) was 8 bytes ago --- */
	uint16_t                   hb_record_len;        /*   264     2 */
	uint8_t                    finished;             /*   266     1 */

	/* XXX 5 bytes hole, try to pack */

	DetectEngineState *        de_state;             /*   272     8 */
	uint16_t                   events;               /*   280     2 */

	/* XXX 6 bytes hole, try to pack */

	AppLayerDecoderEvents *    decoder_events;       /*   288     8 */

	/* size: 296, cachelines: 5, members: 10 */
	/* sum members: 281, holes: 3, sum holes: 15 */
	/* last cacheline: 40 bytes */
};
</pre>